### DISCUSSION TITLE
Is it possible to view a list of all the elements that have been blocked?

### DISCUSSION THREAD

**Discussion by K4LCIFER** [ClearURLs/Addon#171] [Q&A] [answered]
If you click on the ClearURLs addon icon, you get a quick rundown of the statistics of what has been blocked in pure numbers: total number of elements, blocked elements, and percentage, but is it possible to see what was actually removed for the page in a list somewhere? I want to see what parts of the URL have been removed, and what page elements.

**Comment by KevinRoebert** [comment, marked as answer]
Yes, activate the logging. Then a new entry appears in the popup. By clicking on the button, an overview opens in which all operations of ClearURLs are listed in tabular form.
![Bildschirmfoto 2022-02-01 um 22 29 47](https://user-images.githubusercontent.com/28713602/152054558-a88a8607-1053-40d3-b55a-29058f5051d2.png)

**Reply by K4LCIFER** [reply]
Beautiful! Thanks! Although I am curious, why does it recommend to not have it enabled? Is it because it can end up taking up a lot of space if left on for long periods of time? Or perhaps because its a secondary log of browsing history?

**Reply by KevinRoebert** [reply]
The recommendation is given because, on the one hand, a lot of memory is used and the addon, as well as the browser, can become slower as a result. On the other hand, for privacy reasons; the most secure way against misuse is data that has not been collected in the first place.

**Reply by K4LCIFER** [reply]
> The recommendation is given because, on the one hand, a lot of memory is used and the addon, as well as the browser, can become slower as a result. On the other hand, for privacy reasons; the most secure way against misuse is data that has not been collected in the first place.

Makes sense! Thank you very much for the expedient replies.
